cabal.project

```diff
@@ -110,6 +110,8 @@ constraints:
   -- ones they reuse the one from 'some', but there isn't e.g. a proper version
   -- constraint from dependent-sum-template (which is the library we actually use).
   , dependent-sum > 0.6.2.0
+  , algebraic-graphs < 0.7
```

Comment:
I can't figure out why this upper bound is needed, which package depends on it? None of the packages in cardano-ledger directly depend on it, neither does the cardano-prelude. I wouldn't care if it was a lower bound, but upper bound can be problematic in the future.

Reply:
It's a `plutus` dependency.  The bound is needed because `plutus` is missing the upper bound and won't compile with the latest package version.

Reply:
I see, plutus repo hasn't bumped the index yet, that's why they don't have that upper bound set. 

Makes sense, thanks.